tests/data/schema/cwfs.yaml

```diff
@@ -0,0 +1,30 @@
+$schema: http://json-schema.org/draft-07/schema#
+type: object
+properties:
+  find_target:
+    type: object
+    additionalProperties: false
+    required:
+      - az
+      - el
+      - mag_limit
+    description: >-
+        Optional configuration section. Find a target to perform CWFS in the given
+        position and magnitude range. If not specified, the step is ignored.
+    properties:
+      az:
+        type: number
+        description: Azimuth (in degrees) to find a target.
+      el:
+        type: number
+        description: Elevation (in degrees) to find a target.
+      mag_limit:
+        type: number
+        description: Minimum (brightest) V-magnitude limit.
+    program:
+        type: string
+        description: Name of the program these observations are part of.
+required:
+  - find_target
+  - program
```

Comment:
I am surprised that this works. `program` is not at the same indentation level as `find_targets`. Like this `program` seems to be a property of `find_target` instead of another item in the schema. In `cwfs.py` the indentation is correct so please fix it here.

Reply:
oh, I think the multiple levels confused you here. The properties above (that `az`, `el`, `mag_limit` belong to) are from `find_target`. So `program` belongs to the higher level, e.g., same as `find_target`.

Reply:
As commented on Slack, the indentation of `program` is wrong.